### DISCUSSION TITLE
ForBo7 // Salman Naqvi - Adding Subscriptions to a Quarto Site

### DISCUSSION THREAD

**Discussion by giscus** [ForBo7/forbo7.github.io#6] [Giscus Comments]
# ForBo7 // Salman Naqvi - Adding Subscriptions to a Quarto Site

A no nonsense, to the point guide to implementing subscriptions in your Quarto site.

https://forbo7.github.io/forblog/posts/7_blog_subscriptions.html

**Comment by kamurri** [comment]
Where can we find resources for handling the backend with JavaScript and other technologies in the HTML & JS option?

**Reply by ForBo7** [reply]
As I haven't dabbled in JS before, I'm unable to recommend any resources. Though, if you do a quick search online, you should be able to find some guides and tutorials regarding how to use JS to implement a form.

From my current understanding of JS, however, JS is simply used to process whatever has been input client side. After the input has been processed, it will need to be sent to a server, where it will be stored in a database. This will require a stack comprising other tools. For the server and database part, I assume using a third party solution will be the easiest, unless you would like to host your own server!

**Comment by kirkpsmith** [comment]
Hi, awesome post! I've implemented something similar on two Quarto blogs I've set up: https://dualpower.supply/subscribe.html and https://fbrc.dev/subscribe.html 

I am using a self-hosted instance of https://listmonk.app/ and https://github.com/ElliotKillick/rss2newsletter together, though you don't have to use them together. Zero cost for me except a bit of time to set up, though I haven't had to fiddle with it much after! Just be sure to enable the captcha feature on Listmonk if you use it.
